src/public/CesiumOmniverse.cpp

```diff
@@ -185,6 +185,10 @@ class CesiumOmniversePlugin final : public ICesiumOmniverseInterface {
 #endif
     }
 
+    void clearAccessorCache() override {
+        _pContext->clearAccessorCache();
+    }
```

Comment:
It wasn't clear to me that it it is `noexcept`; CesiumAsync::ICacheDatabase::clearAll() is not marked `noexcept`. I suppose that we could wrap a try / catch block inside.

Reply:
Hm, I was under the impression that everything in native was `noexcept` since it's expected to work in environments where extensions are disabled (e.g. Unreal). I wonder if that was an oversight or intentional.

In any case, the reason these functions are `noexcept` is because of [Carbonite plugin ABI rules](http://omniverse-docs.s3-website-us-east-1.amazonaws.com/carbonite/164.0-pre/docs/ABI.html#exceptions). CesiumOmniverse is meant to be a Carbonite plugin, though a few C++ runtime types have snuck into the interface, which should eventually be fixed (cc #3).